### DISCUSSION TITLE
Parsing Route Parameters

### DISCUSSION THREAD

**Discussion by iamMHZ** [encode/django-rest-framework#9041] [Question & Answer] [answered]
Suppose we have this urlpattern `/<int:school_id>/courses ` and we wanna access `school` in the serializer ( I know that we can do `self.context['view'].kwargs['school_id']`)

But would it be possible to have it as a serializer field?
Something like this:

```
class CourseSerializer(serializers.Serializer):
    school = serializers.RouteParameter(name='school_id', queryset=School.objects.all())
    ....

```

**Comment by iamMHZ** [comment]
@auvipy
@tomchristie
As far as I know, It isn't supported in the DRF and I know it is more of a sugar syntax. Can I do a PR on it?

**Reply by auvipy** [reply, marked as answer]
that is not needed I guess as you can implement it in your projects :)
